swap_meet/vendor.py

```diff
@@ -1,2 +1,107 @@
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else: 
+            return False
```

Comment:
Prefer to invert the sense of the comparison to reorder the logic as a guard clause, emphasizing the main logic (removing an item) by leaving it unindented
```py
        if item not in self.inventory:
            return False

        self.inventory.remove(item)
        return item
```